src/components/Providers/TripProvider.jsx

```diff
@@ -14,13 +12,11 @@ export const useTrip = () => {
 }
 
 const TripProvider = ({ timeserie, children }) => {
-  const cleanedGeojson = removeAggregationFromTimeseries(timeserie)
-
   const value = useMemo(
     () => ({
-      timeserie: cleanedGeojson
+      timeserie
     }),
-    [cleanedGeojson]
+    [timeserie]
   )
```

Comment:
I'm not sure I understand this provider: it basically just returns the timeserie that is provided by prop? What for?

Reply:
parce que c'est un parent qui passe la prop... mais ensuite ça permet aux enfants d'y accéder facilement. Et typiquement ici, le parent est sur une route